src/com/untamedears/PrisonPearl/PrisonPearlPlugin.java

```diff
@@ -132,6 +133,8 @@ public void onEnable() {
 		load(portalman, getPortaledPlayersFile());
 		broadcastman = new BroadcastManager();
 		combatTagManager = new CombatTagManager(this.getServer(), log);
+		wbManager = new WorldBorderManager(this);
+		load(wbManager, getWhiteListedCordsFile());
```

Comment:
"getWhiteListedCordsFile()" -- pretty sure you mean to use method "getWhiteListedLocationsFile()" from line 431, yes?

Does this compile?